charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.1.1
+version: 0.2.0
```

Comment:
This change is no-op for end user. Let's call it `0.1.2`

Reply:
Okay, makes sense.